diff --git a/Makefile b/Makefile
index 622933b..a77efb2 100644
--- a/Makefile
+++ b/Makefile
@@ -230,7 +230,7 @@ $(bin/screensaver.so): $(build/web-greeter)
 $(bin_local/web-greeter): build_install_root $(resources.py) $(bin/screensaver.so)
 	@rm -rf "${INSTALL_PREFIX}/lib/web-greeter/*"
 	@cp -R "${BUILD_DIR}/web-greeter"/* "${INSTALL_PREFIX}/lib/web-greeter"
-	@printf "#!/usr/bin/env bash\npython3 ${DESTDIR_PREFIX}/lib/web-greeter \$$@" > \
+	@printf "#!/usr/bin/env bash\npython3 /usr/lib/web-greeter \$$@" > \
 		"${bin_local/web-greeter}"
 	@chmod +x "${bin_local/web-greeter}"
 	@echo "✔ web-greeter binary copied"
